pallets/pallet-asset-switch/src/switch.rs

```diff
@@ -21,24 +21,65 @@ use scale_info::TypeInfo;
 use sp_core::RuntimeDebug;
 use xcm::{VersionedAssetId, VersionedMultiAsset, VersionedMultiLocation};
 
+/// Input information used to generate a `SwitchPairInfo`.
+#[derive(Encode, Decode, TypeInfo, RuntimeDebug, Clone)]
+pub struct NewSwitchPairInfo<AccountId> {
```

Comment:
The reason why this struct exist is because I wanted to hide one of the fields of the `SwitchPairInfo` struct, specifically the remote balance, as it's not supposed to be used directly. What would you suggest we do instead?

Reply:
Yeah difficult. I think it is fine.